app/src/test/java/io/apicurio/registry/storage/RegistryStorageSmokeTest.java

```diff
@@ -0,0 +1,104 @@
+package io.apicurio.registry.storage;
+
+import io.apicurio.registry.rest.beans.ArtifactType;
+import io.apicurio.registry.storage.impl.jdbc.Jdbc;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.util.Collections;
+import java.util.SortedSet;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@QuarkusTest
+public class RegistryStorageSmokeTest {
+
+    private static Logger log = LoggerFactory.getLogger(RegistryStorageSmokeTest.class);
+
+    @Inject
+    @Jdbc
```

Comment:
This can be made @Current ?
So same test can be re-used in other impls ...

Reply:
It does not work properly. If I use the annotation, the producer provides the in-memory implementation, which also fails the tests. This a part of a bigger issue, since we don't want to run the tests on a single implementation but on all of them.

Reply:
I think we should solve this separately.